sentry-android-core/proguard-rules.pro

```diff
@@ -24,6 +24,9 @@
 -keep class * implements com.google.gson.JsonSerializer
 -keep class * implements com.google.gson.JsonDeserializer
 
+# Prevent proguard from minifying exception type names
+-keep public class * extends java.lang.Exception
```

Comment:
what's about
`-keep class * extends java.lang.RuntimeException` as well?
so we cover checked and unchecked exceptions.

I'd also remove `public` as it could be an internal exception right

Reply:
Makes sense.